aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Form, FormField, RadioButtonGroup } from 'grommet';
+
+const powerRegulationOptions = [
+  'Dynamic power savings mode',
+  'Static low power mode',
+  'Static high performance mode',
+  'OS control mode',
+];
+
+export const RadioButtonGroupValidationExample = () => {
+  const [powerRegulation, setPowerRegulation] = useState();
+  const [message, setMessage] = useState({
+    error: 'Required information.',
+    info: `To ensure your product is configured correctly, select 
+    a Power Regulation option to resolve this error.`,
+  });
+
+  const onChange = value => {
+    setPowerRegulation(value);
+    if (typeof powerRegulation === 'undefined') {
+      setMessage();
+    }
+  };
+
+  return (
+    <Form>
```

Comment:
When I click on one of these radio buttons, the page goes to an error screen.

Reply:
will fix. Thanks.

Reply:
Fixed.